configs/trainer/dropout_scheduler/triangle.yaml

```diff
@@ -0,0 +1,4 @@
+dropout_type: "triangle"
+start_dropout_p: 0.0
+end_dropout_p: 0.1
```

Comment:
only comment is maybe rename to like dropout_peak/ dropout_trough or sth

Reply:
cos its not really start and finish

Reply:
you're right. Let me rename.